handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -115,10 +115,8 @@ public abstract class ReferenceCountedOpenSslContext extends SslContext implemen
 
     static final boolean SERVER_ENABLE_SESSION_CACHE =
             SystemPropertyUtil.getBoolean("io.netty.handler.ssl.openssl.sessionCacheServer", true);
-    // session caching is disabled by default on the client side due a JDK bug:
```

Comment:
Let's update this comment then stating it is now enabled from being disabled previously. Also, let's add a logger for this in the `static` block so people can explicitly know that it's turned on now.

Reply:
honestly I think there is no need for a comment or static logger